mlflow/utils/file_utils.py

```diff
@@ -21,11 +21,13 @@
 except ImportError:
     from yaml import SafeLoader as YamlSafeLoader, SafeDumper as YamlSafeDumper
 
+
```

Comment:
I think the changes in this file belong in `model_utils`. Most of these utils either don't operate on files (e.g. `validate_code_paths`) or rely on other `file_utils` methods to interact with files without adding any new file-specific logic (e.g. `_copy_code_paths`).

Reply:
Can we also add docstrings to the new utility methods?